src/main/java/com/google/sps/CapstoneAuth.java

```diff
@@ -10,74 +10,111 @@
 import com.google.firebase.auth.FirebaseAuth;
 import com.google.firebase.auth.FirebaseToken;
 import com.google.firebase.auth.FirebaseAuthException;
-import com.google.appengine.api.datastore.DatastoreServiceFactory;
-import com.google.appengine.api.datastore.DatastoreService;
-import com.google.appengine.api.datastore.Entity;
-import com.google.appengine.api.datastore.KeyFactory;
+import com.google.cloud.datastore.Datastore;
+import com.google.cloud.datastore.DatastoreOptions;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.KeyFactory;
+import com.google.cloud.datastore.ListValue;
+import com.google.cloud.datastore.StringValue;
+import com.google.cloud.datastore.Value;
 
 import com.google.auth.oauth2.GoogleCredentials;
 
 public final class CapstoneAuth {
 
     private static CapstoneAuth currentInstance;
-    private DatastoreService datastoreService;
+
+    private final Datastore datastore;
+    private final KeyFactory keyFactory;
 
     private CapstoneAuth() {
         try {
             InputStream credentialInputStream = getClass().getResourceAsStream(
                 "/chap-2020-capstone-firebase-adminsdk-93l38-698b686069.json");
 
             FirebaseOptions options = new FirebaseOptions.Builder()
-                .setCredentials(GoogleCredentials.fromStream(credentialInputStream))
+                .setCredentials(
+                    GoogleCredentials.fromStream(credentialInputStream))
                 .setDatabaseUrl("https://chap-2020-capstone.firebaseio.com/")
                 .build();
 
             FirebaseApp.initializeApp(options);
-
-            datastoreService = DatastoreServiceFactory.getDatastoreService();
-
-            addTestChatroom();
         } catch (IOException e) {
             e.printStackTrace();
         }
+
+        datastore = DatastoreOptions.getDefaultInstance().getService();
+        keyFactory = datastore.newKeyFactory().setKind("CHAT_ROOM");
+
+        addTestChatroom();
     }
 
     // To remove method on deploy
```

Comment:
Use the format `// TODO(<ldap>): message`

Example: `// TODO(lopezalexander): Remove method on deploy.`